scripts/configure_invokeai.py

```diff
@@ -587,31 +557,63 @@ def get_root(root:str=None)->str:
             match = re.search('--root\s*=?\s*"?([^"]+)"?',l)
             if match:
                 root = match.groups()[0]
-    root = root or '.'
-    return root.strip()
+                root = root.strip()
+    return root
+
+#-------------------------------------
+def select_root(yes_to_all:bool=False):
+    default = os.path.expanduser('~/invokeai')
+    if (yes_to_all):
+        return default
+    completer.set_default_dir(default)
+    completer.complete_extensions(())
+    completer.set_line(default)
+    return input(f"Select a directory in which to install InvokeAI's models and configuration files [{default}]: ")
+
+#-------------------------------------
+def select_outputs(root:str,yes_to_all:bool=False):
+    default = os.path.normpath(os.path.join(root,'outputs'))
+    if (yes_to_all):
+        return default
+    completer.set_default_dir(os.path.expanduser('~'))
+    completer.complete_extensions(())
+    completer.set_line(default)
+    return input('Select the default directory for image outputs [{default}]: ')
 
 #-------------------------------------
-def initialize_rootdir(root:str):
+def initialize_rootdir(root:str,yes_to_all:bool=False):
     assert os.path.exists('./configs'),'Run this script from within the top level of the InvokeAI source code directory, "InvokeAI"'
-    print(f'** INITIALIZING INVOKEAI ROOT DIRECTORY **')
-    print(f'Creating a directory named {root} to contain InvokeAI models, configuration files and outputs.')
-    print(f'If you move this directory, please change its location using the --root option in "{Globals.initfile},')
-    print(f'or set the environment variable INVOKEAI_ROOT to the new location.\n')
-    for name in ('models','configs','outputs','scripts','frontend/dist'):
+    
+    print(f'** INITIALIZING INVOKEAI RUNTIME DIRECTORY **')
+    root = root or select_root(yes_to_all)
+    outputs = select_outputs(root,yes_to_all)
+    Globals.root = root
+        
+    print(f'InvokeAI models and configuration files will be placed into {root} and image outputs will be placed into {outputs}.')
+    print(f'\nYou may change these values at any time by editing the --root and --output_dir options in "{Globals.initfile}",')
+    print(f'You may also change the runtime directory by setting the environment variable INVOKEAI_ROOT.\n')
+    for name in ('models','configs','scripts','frontend/dist'):
         os.makedirs(os.path.join(root,name), exist_ok=True)
     for src in ('configs','scripts','frontend/dist'):
         dest = os.path.join(root,src)
         if not os.path.samefile(src,dest):
             shutil.copytree(src,dest,dirs_exist_ok=True)
+    os.makedirs(outputs)
```

Comment:
The script crashes if the `outputs` exists. Needs `exist_ok=True`.

```suggestion
    os.makedirs(outputs, exist_ok=True)
```